apps/cave/components/UserDashboard/Summary/Staking/LiquidStakingSnapshot.tsx

```diff
@@ -0,0 +1,136 @@
+import { AirdropClaimContract } from '@concave/core'
+import { SpinIcon, SpinnerIcon } from '@concave/icons'
+import { Flex, Spinner } from '@concave/ui'
+import { airdropToken, getAirdropQ4ClaimableAmount } from 'components/Airdrop/Q4/airdrop'
+import { getAirdropSpecialClaimableAmount } from 'components/Airdrop/special/airdrop'
+import { useFilterByRange } from 'components/NftFilters/Filters/hooks/useFilterByRange'
+import { useFilterByStakePool } from 'components/NftFilters/Filters/hooks/useFilterByStakePool'
+import { usePositionSorter } from 'components/NftFilters/Sorters/hooks/useNftSort'
+import { useStakePositions } from 'components/StakingPositions/DashboardBody/DashBoardState'
+import { FilterContainer } from 'components/StakingPositions/DashboardBody/FilterContainer'
+import { SnapshotLineChart } from 'components/UserDashboard/SnapshotLineChart'
+import { SnapshotTextCard } from 'components/UserDashboard/SnapshotTextCard'
+import { useStakeSettings } from 'contexts/PositionsFilterProvider'
+import { AirdropSeason } from 'hooks/useAirdropSeason'
+import { useCurrentSupportedNetworkId } from 'hooks/useCurrentSupportedNetworkId'
+import { concaveProvider } from 'lib/providers'
+import { useState } from 'react'
+import { useQuery } from 'react-query'
+import { useAccount } from 'wagmi'
+import { UserPositionCard } from '../../../StakingPositions/LockPosition/Card/UserPositionCard'
+import { DataTable } from '../../DataTable'
+import { DataTableCard } from '../../DataTableCard'
+import { SnapshotCard } from '../../SnapshotCard'
+import { SnapshotText } from '../../SnapshotText'
+import { stakechartdata } from '../dummyChartData'
+
+const useAirdropOverview = (address:string, chainId) => {
+  return useQuery(['AirdropClaimOverview', chainId], async () => {
+    const provider = concaveProvider(chainId)
+    const seasons = [{
+        claimed: await new AirdropClaimContract(provider, 'special').claimed(address),
+        amount: getAirdropSpecialClaimableAmount(address) || 0
+      },{
+        claimed: await new AirdropClaimContract(provider, 'Q4').claimed(address),
```

Comment:
I'm not sure did you test if awaiting one after the other get's bundled into a single multicall or maybe we should do it like
```ts
// start both promises without blocking 
const claimedSpecial = new AirdropClaimContract(provider, 'special').claimed(address)
const claimedQ4 = new AirdropClaimContract(provider, 'Q4').claimed(address) 
// then await their response to return
const seasons = [{
        claimed: await claimedSpecial,
        amount: getAirdropSpecialClaimableAmount(address) || 0
      },{
        claimed: await claimedQ4,
```

Reply:
are you right